src/zarr/api/asynchronous.py

```diff
@@ -251,8 +312,11 @@ async def open(
         return await open_group(store=store_path, zarr_format=zarr_format, **kwargs)
 
 
-async def open_consolidated(*args: Any, **kwargs: Any) -> AsyncGroup:
-    raise NotImplementedError
+async def open_consolidated(*args: Any, use_consolidated: bool = True, **kwargs: Any) -> AsyncGroup:
+    """
+    Alias for :func:`open_group` with ``use_consolidated=True``.
+    """
+    return await open_group(*args, use_consolidated=use_consolidated, **kwargs)
```

Comment:
My goal here is to avoid a user accidentally passing `use_consolidated` in `**kwargs` and us silently overwriting it. I'll update this to raise if `use_consolidated` isn't True.